access/slack/Makefile

```diff
@@ -16,6 +16,8 @@ RELEASE_MESSAGE := "Building with GOOS=$(OS) GOARCH=$(ARCH)."
 
 VERSRC = version.go gitref.go
 
+IMG ?= hashtable/teleport-access-slack:latest
```

Comment:
This should probably be `quay.io/gravitational/teleport-access-slack:$(Version)`, which is in line with the teleport docker image we publish. @russjones is this correct?

For testing you could still use your own docker hub account.

Reply:
Yes